benchmark/how-to-run-tpcc.md

```diff
@@ -137,15 +139,94 @@ loadWorkers=32  # The number of concurrent workers that load data.
     ./runSQL.sh props.mysql sql.mysql/indexCreates.sql
     ```
 
-3. Run the following script to load data:
+### Use BenchmarkSQL to load data directly
+
+Run the following script to load data:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+./runLoader.sh props.mysql
+```
+
+This process might last for several hours depending on the machine configuration.
+
+### Use TiDB Lightning to load data
+
+The amount of imported data increases with the number of warehouses. When you import more than 1000 warehouse data, you can generate csv files using BenchmarkSQL, and then quickly import the csv files through TiDB Lightning (hereinafter referred to as Lightning). The csv files can be reused multiple times, saving the time required for each generation.
+
+1. Modify the BenchmarkSQL configuration file.
+
+    The csv file of one warehouses requires 77 MB of disk space. To ensure sufficient disk space, add a line to the `benchmarksql/run/props.mysql` file:
+
+    ```text
+    fileLocation=/home/user/csv/  # Absolute path to directory where the csv files are stored
+    ```
+
+    The csv file name should adhere to the Lightning naming rules, namely `{database}.{table}.csv`, because you use TiDB Lightning to load data. Here you can modify the above configuration to:
+
+    ```text
+    fileLocation=/home/user/csv/tpcc.  # Absolute path to directory where the csv files are stored + file name prefix (database)
+    ```
+
+    In this case, the format of the csv file name is similar to `tpcc.bmsql_warehouse.csv`.
+
+2. Generate the csv file.
 
     {{< copyable "shell-regular" >}}
 
     ```shell
     ./runLoader.sh props.mysql
     ```
 
-This process might last for several hours depending on the machine configuration.
+3. Use Lightning to load data.
+
+    To load data using Lightning, see [TiDB Lightning Deployment](/reference/tools/tidb-lightning/deployment.md). This section will show you how to deploy Lightning using tidb-ansible to load data.
+
+    1. Edit `inventory.ini`.
+
+        It is recommended to specify the deployed IP, the port and the directory manually to avoid anomalies caused by conflicts. For the disk space of `import_dir`, see [TiDB Lightning Deployment](/reference/tools/tidb-lightning/deployment.md). `data_source_dir` refers to the directory where the csv files are stored, as mentioned above.
```

Comment:
```suggestion
        It is recommended to manually specify the deployed IP address, the port, and the deployment directory to avoid anomalies caused by conflicts. For the disk space of `import_dir`, see [TiDB Lightning Deployment](/reference/tools/tidb-lightning/deployment.md). `data_source_dir` refers to the directory where the CSV files are stored as mentioned before.
```